scripts/dev.js

```diff
@@ -37,9 +37,10 @@ const removeArtifacts = async () => {
 const dev = async (maybeInit) => {
   const isInit = !fs.existsSync(path.join(TOOLBOX_ROOT, 'updateSchema.js')) || maybeInit
   const redis = new Redis(process.env.REDIS_URL)
+  const toolboxPromise = compileToolbox()
```

Comment:
always rebuild the toolbox in order to get a fresh schema in order to rebuild the schema. takes a little longer to start up, but guarantees that the graphql schema is correct